javascript-methods/main.js

```diff
@@ -0,0 +1,59 @@
+const num1 = 5;
+const num2 = 8;
+const num3 = 7;
+const maximumValue = Math.max(num1, num2, num3);
+console.log('maximumValue:', maximumValue);
+
+const heroes = ['spider-man', 'batman', 'venom', 'John Cena'];
+const randomNumber = Math.random(heroes) * heroes.length;
```

Comment:
`Math.random()` does not take any arguments. You should originally be calling this on it's own and assigning it to `randomNumber`, and then that's when you'd multiple `randomNumber * heroes.length` and reassign it back to `randomNumber`